### ISSUE TITLE
Ethereum derivation path for Ledger Live calculated incorrectly

### ISSUE THREAD

**Issue by vevnin** [LedgerHQ/ledgerjs#307]
In case when user creates multiple Ethereum accounts in Ledger Live 1.4.1 derivation path for each of them is set in pattern described in [BIP 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) incrementing account field:
- 44'/60'/0'/0/0
- 44'/60'/1'/0/0
- 44'/60'/2'/0/0

Web3-subprovider now supports 44'/60'/0'/0/0 path, but when it performs scan for addresses it generates paths incrementing address_index field instead:
- 44'/60'/0'/0/0
- 44'/60'/0'/0/1
- 44'/60'/0'/0/2

This way user of dapp where Web3-subprovider is used can see only first address from Ledger Live and can't see the rest of claimed addresses.

**Comment by vevnin** [comment]
#273

**Comment by gre** [comment]
Yes as you said, the BIP44 is favorized in current implementation of Ledger Live derivation. This is not likely going to change.

However, Ledger Live **does** scan for `44'/60'/0'/0/{x}` accounts, so these should be discovered! meaning you should be able to move eths to these accounts and it will work in Ledger Live.

Maybe in the future, we can offer in Ledger Live a preference to chose the derivation. If someone want to offer a new Settings in Ethereum currency settings, be my guest!

**Comment by vevnin** [comment]
@gre The issue is not that Ledger Live doesn't detect 44'/60'/0'/0/{x} accounts. It's rather related to web3-subprovider and how it performs account scan.

In case if account is created first in Ledger Live, it gets created with 4'/60'/{x}'/0/0 path. web3-subprovider scans using  44'/60'/0'/0/{x} pattern and it **does not** detect any accounts created in Ledger Live besides first one. It basically gives you different set of addresses starting from second one.

This way users see mismatch between accounts displayed in Ledger Live and in dapp using web3-subprovider.

**Comment by gre** [comment]
OH yeah sorry! I thought you were mentionning an external library!

you are correct.

**Comment by gre** [comment]
this is now fixed (version `5.2.0`)

you can do this:

```
paths: ["44'/60'/x'/0/0"]
```

or


```
paths: ["44'/60'/0'/x"]
```

and even


```
paths: ["44'/60'/x'/0/0", "44'/60'/0'/x"]
```

which will yield: "44'/60'/0'/0/0", "44'/60'/0'/0", "44'/60'/1'/0/0", "44'/60'/0'/1", ...

that last one is the default.
